assets/js/index.js

```diff
@@ -7,3 +7,72 @@ var formFocus = toolkit.formFocus;
 
 helpers.documentReady(progressiveReveal);
 helpers.documentReady(formFocus);
+
+var $ = require('jquery');
+var typeahead = require('typeahead-aria');
+var Bloodhound = require('typeahead-aria').Bloodhound;
+
+typeahead.loadjQueryPlugin();
+
+$('.typeahead').each(function applyTypeahead() {
+  var $el = $(this);
+  var $parent = $el.parent();
+  var attributes = $el.prop('attributes');
+  var $input = $('<input/>');
+  var selectedValue = $el.val();
+  var typeaheadList = $el.find('option').map(function mapOptions() {
```

Comment:
@JoeChapman making this module es6 would require adding a babel build step. Simplest way would be to use the babelify browserify-transform. Would also need to add a .babelrc and enable desired features